Makefile

```diff
@@ -1,49 +1,23 @@
-# Makefile used to create packages for mixer-tools. It doesn't assume
-# that the code is inside a GOPATH, and always copy the files into a
-# new workspace to get the work done. Go tools doesn't reliably work
-# with symbolic links.
-#
-# For historical purposes, it also works in a development environment
-# when the repository is already inside a GOPATH.
 include Makefile.bats
 
 .NOTPARALLEL:
 
 VERSION=6.1.4
 GO_PACKAGE_PREFIX := github.com/clearlinux/mixer-tools
 
-.PHONY: gopath
-
-# Strictly speaking we should check if it the directory is inside an
-# actual GOPATH, but the directory structure matching is likely enough.
-ifeq (,$(findstring ${GO_PACKAGE_PREFIX},${CURDIR}))
-LOCAL_GOPATH := ${CURDIR}/.gopath
-export GOPATH := ${LOCAL_GOPATH}
-gopath:
-	@rm -rf ${LOCAL_GOPATH}/src
-	@mkdir -p ${LOCAL_GOPATH}/src/${GO_PACKAGE_PREFIX}
-	@cp -af * ${LOCAL_GOPATH}/src/${GO_PACKAGE_PREFIX}
-	@echo "Prepared a local GOPATH=${GOPATH}"
-else
-LOCAL_GOPATH :=
-GOPATH ?= ${HOME}/go
-gopath:
-	@echo "Code already in existing GOPATH=${GOPATH}"
-endif
-
 .PHONY: build install clean check
 
 .DEFAULT_GOAL := build
 
 
-build: gopath
+build:
 	go install -ldflags="-X ${GO_PACKAGE_PREFIX}/builder.Version=${VERSION}" ${GO_PACKAGE_PREFIX}/mixer
 	go install ${GO_PACKAGE_PREFIX}/mixin
 	go install ${GO_PACKAGE_PREFIX}/swupd-extract
 	go install ${GO_PACKAGE_PREFIX}/swupd-inspector
 	go install ${GO_PACKAGE_PREFIX}/mixer-completion
 
-install: gopath
+install:
```

Comment:
`make install` succeeds when I set a GOPATH, but it fails when the GOPATH is unset.

By default, the go environment uses a GOPATH of `$HOME/go`, so `go install.....` will create the executable at `$HOME/go/bin/mixer`. The `install -m.....` command is not a go command, so it does not use the same default GOPATH. As a result, the executable is created at `$HOME/go/bin/mixer`, but the when installing `/bin/mixer` is used as the source path.

I suggest adding a default value for GOPATH: `GOPATH ?= ${HOME}/go`

Reply:
@jwakre I've made the suggested change including doing a `build` while installing to take care of the sudo requirement. Let me know if this works..